packages/material-ui/src/Select/SelectInput.test.js

```diff
@@ -286,21 +275,29 @@ describe('<SelectInput />', () => {
   });
 
   describe('prop: autoWidth', () => {
+    let clock;
```

Comment:
Is this needed at all? I have tried the test without the fake timer. It passes.

Reply:
Looks like some code I left in accidentally when getting the tests to pass. Nice spot :+1: